vclib-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidc/AuthenticationRequestParameters.kt

```diff
@@ -240,35 +240,3 @@ data class AuthenticationRequestParameters(
         }
     }
 }
-
-// restricted to internal as there is no reason to make this public for now
```

Comment:
Because with this class you don't see the serial name of the property at a glance (which is quite useful for OID obejcts). Additionally, it is different from all other classes then. Also, nobody needs the serial names.